### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by The-Young-Programmer** [Samuel-de-Oliveira/Helen#5] [General]
Hi, I'm back have being away for some time.
Let countinue with the project. Where do you need me?

**Comment by Samuel-de-Oliveira** [comment]
Oh hello dude, for while I'm busy in other things.

But I have question: Do you speak other language than the English? If you do, can you make a file translated of [this one](https://github.com/Samuel-de-Oliveira/Helen/blob/main/Tutorial/README.md)? Don't you worry about the HC-SR04 and 05 README for a while I will organize them soon.

**Reply by The-Young-Programmer** [reply]
Okay, I can translate to Mandarin (Language of China 🇨🇳) and hanguk-eo ("Korean language 🇰🇷").
